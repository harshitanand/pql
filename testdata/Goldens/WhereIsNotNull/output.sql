SELECT * FROM "Tokens" WHERE ("Value") IS NOT NULL;
