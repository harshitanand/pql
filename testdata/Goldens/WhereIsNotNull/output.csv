Kind,TokenConstant
1,TokenIdentifier
2,TokenQuotedIdentifier
3,TokenNumber
4,TokenString
5,TokenAnd
6,TokenOr
7,TokenPipe
-1,TokenError
